packages/shared/components/popups/TokenInformationPopup.svelte

```diff
@@ -58,19 +58,37 @@
             <TextHint warning text={localize('popups.tokenInformation.verificationWarning')} />
         {/if}
         <div class="w-full flex-col space-y-2">
-            <KeyValueBox
-                keyText={localize('popups.tokenInformation.tokenMetadata.name')}
-                valueText={asset?.metadata?.name}
-            />
-            <KeyValueBox
-                keyText={localize('popups.tokenInformation.tokenMetadata.tokenId')}
-                valueText={truncateString(asset?.id, 15, 15, 3)}
-            />
+            <CopyableBox value={asset?.metadata?.name} classes="justify-between">
+                <Text fontSize="14" lineHeight="5" color="gray-600" darkColor="gray-500" classes="mr-4">
+                    {localize('popups.tokenInformation.tokenMetadata.name')}
+                </Text>
+                <div class="truncate">
+                    <Text fontSize="14" lineHeight="5" color="gray-600" darkColor="gray-500" classes="truncate">
+                        {asset?.metadata?.name}
+                    </Text>
+                </div>
+            </CopyableBox>
+            <CopyableBox value={asset?.id} classes="justify-between">
+                <Text fontSize="14" lineHeight="5" color="gray-600" darkColor="gray-500" classes="mr-4">
+                    {localize('popups.tokenInformation.tokenMetadata.tokenId')}
+                </Text>
+                <div class="truncate">
+                    <Text fontSize="14" lineHeight="5" color="gray-600" darkColor="gray-500" classes="truncate">
+                        {truncateString(asset?.id, 15, 15, 3)}
+                    </Text>
+                </div>
+            </CopyableBox>
```

Comment:
Can we make the KeyValue box have a copyable property and stick all this stying etc in the key value box so we can reuse this.